cartridges/int_klaviyo/cartridge/controllers/Klaviyo.js

```diff
@@ -76,5 +76,28 @@ var StartedCheckoutEvent = function () {
     responseUtils.renderJSON({ success: true });
 };
 
+
+/* called via ajax when the email or sms subscribe checkboxes are clicked.  KLSubscribe session vars picked up in Klaviyo-appended Order-Confirm controller */
```

Comment:
Tiny nit: Do we want to change the language here for SG - not the Order-Confirm controller but the COSummary controller

Reply:
Good catch. Thank you.